stacks_queues/stack.py

```diff
@@ -21,18 +23,27 @@ def pop(self):
             The Stack is empty.
             returns None
         """
-        pass
+        if self.store == None:
+            raise StackEmptyException("Stack is empty")
+        else:
+            element = self.store.get_first()
+            self.store.remove_first()
+
+        return element
```

Comment:
If you look at the implementation in LinkedList, `remove_first` returns the element it removes!
```suggestion
            return self.store.remove_first()
```